app/directives/statsTable.html

```diff
@@ -0,0 +1,21 @@
+<div class="grid_12">
+    <h5>{{title}}</h5>
+    <table class="two-colours">
+      <tr>
+        <th>Code</th>
+        <th>Name</th>
+        <th>%</th>
+        <th>Count</th>
+      </tr>
+      <tr ng-repeat="item in items">
+        <td>
+          <term-line id="item.key" ng-if="isTerm"></term-line>
+          <strong ng-if="!isTerm">{{item.key}}</strong>
+        </td>
+        <td></td>
+        <td>{{formatNumber(item.percentage)}}</td>
+        <td>{{formatNumber(item.hits)}}</td>
```

Comment:
The $rootScope is missing in the corresponding js directive. As it is now, formatNumber is not known here, meaning no number is displayed at all.